.gitignore

```diff
@@ -31,3 +31,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+#Prop
+.properties
```

Comment:
can you only add the application.properties file in .gitignore, this will disallow us to commit the other two properties file.